### PR TITLE
Added additional generic SNMP trap handlers

### PR DESCRIPTION
**Pull request by CirnoT**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: config/snmptraps.php [hunk 1]
```diff
@@ -58,5 +58,9 @@
         'PowerNet-MIB::outletOn' => \LibreNMS\Snmptrap\Handlers\ApcPduOutletOn::class,
         'PowerNet-MIB::outletReboot' => \LibreNMS\Snmptrap\Handlers\ApcPduOutletReboot::class,
         'SNMPv2-MIB::authenticationFailure' => \LibreNMS\Snmptrap\Handlers\AuthenticationFailure::class,
+        'BRIDGE-MIB::newRoot' => \LibreNMS\Snmptrap\Handlers\BridgeNewRoot::class,
```
Comment (murrant) [inline config/snmptraps.php]: Can you alphabetize these?  otherwise the list is going to be a nightmare ;)

Comment (CirnoT) [reply config/snmptraps.php]: Done

**Comment by CirnoT** [comment]
Sorted it as requested. In the process also noticed dead entry to `AdvaConfigChange` on `ENTITY-MIB::entConfigChange` which has no corresponding file. Didn't notice before as I wrote the code before merging with changes that added Adva stuff.

**Comment by murrant** [comment]
Another good reason to sort them ;)

**Pull request merged**